src/main/java/jaz/Zer.java

```diff
@@ -64,6 +88,25 @@ public Zer(String arg1) {}
 
   public Zer(String arg1, Throwable arg2) {}
 
+  private static boolean isJazZarDisabled() {
+    String allDisabledHooks = System.getProperty("jazzer.disabled_hooks");
```

Comment:
Let's extract the result of the property lookup into a static final constant so that this can benefit from dead code eliminiation.